microsoft-365/security/defender-endpoint/linux-preferences.md

```diff
@@ -125,81 +125,27 @@ Specifies the degree of parallelism for on-demand scans. This corresponds to the
 |**Possible values**|2 (default). Allowed values are integers between 1 and 64.|
 |**Comments**|Available in Microsoft Defender for Endpoint version 101.45.00 or higher.|
 
-#### Exclusion merge policy
+#### Exclusion merge policy (legacy)
 
-Specifies the merge policy for exclusions. It can be a combination of administrator-defined and user-defined exclusions (`merge`) or only administrator-defined exclusions (`admin_only`). This setting can be used to restrict local users from defining their own exclusions.
+Specifies the merge policy for exclusions. It can be a combination of administrator-defined and user-defined exclusions (`merge`) or only administrator-defined exclusions (`admin_only`). This setting can be used to restrict local users from defining their own exclusions. A new field in introduced for merge policy. Refer to - [mergePolicy](https://github.com/AruneemaXD/microsoft-365-docs/blob/user/ardeshmukh/add_global_exclusions_info/microsoft-365/security/defender-endpoint/linux-preferences.md#merge-policy)
```

Comment:
Need to correct link since it is pointing to private branch

Reply:
You can add the relative link something like: [mergePolicy](/microsoft-365/security/defender-endpoint/linux-preferences.md#merge-policy)

Reply:
Updated links wherever required.